webcompat/db/__init__.py

```diff
@@ -46,23 +40,3 @@ def __init__(self, issue_id, summary, url, body):
         self.body = body
 
 IssueBase.metadata.create_all(bind=issue_engine)
```

Comment:
Yeah, also maybe an opportunity to remove it altogether since nobody uses it. ^__^

But that can be a separate discussion (and we will want something like it in the future).